SPECS/grpc/grpc.spec

```diff
@@ -0,0 +1,83 @@
+Summary:        Open source remote procedure call (RPC) framework
+Name:           grpc
+Version:        1.35.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Applications/System
+URL:            https://www.grpc.io
+#Source0:        https://github.com/grpc/grpc/archive/v%{version}/%{name}-%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+# A buildable grpc environment needs functioning submodules that do not work from the archive download
+# To recreate the tar.gz run the following
+#  git clone -b RELEASE_TAG_HERE https://github.com/grpc/grpc
+#  pushd grpc
+#  git submodule update --init
+#  popd
+#  sudo mv grpc grpc-%{version}
+#  sudo tar -cvf grpc-%{version}.tar.gz grpc-%{version}/
+BuildRequires:  git
+BuildRequires:  cmake
+BuildRequires:  gcc
+
+%description
+gRPC is a modern, open source, high-performance remote procedure call (RPC) framework that can run anywhere. It enables client and server applications to communicate transparently, and simplifies the building of connected systems.
+
+%package devel
+Summary:        Development files for grpc
+Requires:       %{name} = %{version}-%{release}
+
+%description devel
+The grpc-devel package contains the header files and libraries
+needed to develop programs that use grpc.
+
+%package plugins
+Summary:        Plugins files for grpc
+Requires:       %{name} = %{version}-%{release}
+Requires:       protobuf-compiler
```

Comment:
What provides `protobuf-compiler`?

Reply:
Changed it to libprotoc, that is provided by the protobuf rpm